BLANK_PAGE_FIX.md

```
@@ -0,0 +1,150 @@
+// AFTER: main.tsx
+// Removed SupabaseProvider - AuthContext already uses hardcoded client
+<I18nProvider>
+  <FeatureFlagProvider>
+    <AuthProvider>
+      {/* ... */}
+    </AuthProvider>
+  </FeatureFlagProvider>
+</I18nProvider>
+```
+
+**Why**: The `AuthContext` already imports and uses the Supabase client from `@/integrations/supabase/client` which has hardcoded credentials. The SupabaseProvider was redundant and causing initialization conflicts.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)